src/services/index.ts

```diff
@@ -660,58 +661,70 @@ export function deployRolesModifier(
   ];
 }
 
-export function deployOptimisticGovernorModule(
+export async function deployOptimisticGovernorModule(
   safeAddress: string,
   chainId: number,
   args: OptimisticGovernorModuleParams
 ) {
-  const type = KnownContracts.OPTIMISTIC_GOVERNOR
   const provider = getProvider(chainId);
 
-  const {
+  const { finder, executor, collateral, bond, rules, identifier, liveness } =
+    args;
+
+  const scaledBond = scaleBondDecimals(collateral, bond).toString();
+  
+  // const type = KnownContracts.OPTIMISTIC_GOVERNOR_FACTORY;
+  const ogFactoryAddress = "0x7fCdC4eE287F6D1F57c25Caa79e7Dc774B44F17c";
```

Comment:
[This address](https://rinkeby.etherscan.io/tx/0x2ae5eb6d9eec25497226e841e015d942cbc4a1dcf48f123bd3264b6e3722e4cf) does not appear to be the result of a deterministic deployment, you'll need to specify the address for each supported network or do a deterministic deployment and update the address.